packages/cssnano/package.json

```diff
@@ -31,10 +31,8 @@
     "repository": "cssnano/cssnano",
     "files": [
         "src",
-        "LICENSE-MIT",
-        "quickstart.js"
+        "LICENSE-MIT"
     ],
-    "tonicExampleFilename": "quickstart.js",
```

Comment:
This is the file that appears on `npm.runkit.com/cssnano` when you click `Try on RunKit` on the npm website. But it just shows an error message. It seems RunKit used to be called tonic.dev, hence the field name.